CONTRIBUTING.md

```diff
@@ -150,9 +150,13 @@ npm run docgen
 
 This updates `docs/` at the top level, where GitHub Pages looks for the docs.
 
-## Update `definitions.json`
+## Updating `definitions.json`
+
+This should almost always be done using [this script](./packages/ripple-binary-codec/tools/generateDefinitions.js) - if the output needs manual intervention afterwards, consider updating the script instead.
 
-Use [this repo](https://github.com/RichardAH/xrpl-codec-gen) to generate a new `definitions.json` file from the rippled source code. Instructions are available in that README.
+1. Clone / pull the latest changes from [rippled](https://github.com/XRPLF/rippled) - Specifically the `develop` branch is usually the right one.
+2. Run `node packages/ripple-binary-codec/tools/generateDefinitions.js path/to/rippled` (assuming you're calling this file from the root directory of xrpl.js).
```

Comment:
```suggestion
2. Run `npm run generateDefinitions path/to/rippled` (assuming you're calling this file from the root directory of xrpl.js).
```

Reply:
nit ^

Reply:
As we discussed previously, that doesn't work.

Reply:
The npm script works for me. You need to be inside the `packages/ripple-binary-codec` directory.